exercises/weighted_voting.livemd

```diff
@@ -97,9 +110,41 @@ defmodule WeightedVoting do
     iex> WeightedVoting.tally([cats: 10, dogs: 20, cats: 20, dogs: 10, birds: 20])
     [birds: 20, cats: 30, dogs: 30]
   """
+
+  # input: keyword list with animal: votes
+  # output: keyword list with animal: total votes, in alphabetical order
+  # solution: 
+  # check the item
+  # if it already exists in the final keyword list, add the number of votes
+  # if it doesn't exist, add a new key and value
+  # when finished with reading, order alphabetically
+  # return
+
   def tally(votes) do
+    Enum.reduce(votes, [], fn {animal, number}, new_list ->
+      if(Enum.member?(new_list, animal))
+
+      List.replace_at(new_list, index, {animal, value})
```

Comment:
We did this today, so I'm not sure if you need feedback anymore.
I believe you solved it using [Keyword.update/4](https://hexdocs.pm/elixir/Keyword.html#update/4)

Reply:
Oh, you are very generous saying I solved it. I was well guided :) But yes, let's close this one and we continue!